examples/embassy-net/build.rs

```diff
@@ -0,0 +1,17 @@
+fn main() {
+    if cfg!(target_os = "linux") {
+        let target_arch = std::env::var("CARGO_CFG_TARGET_ARCH").unwrap();
+
+        // Add back the default linker script.
+        println!("cargo:rustc-link-search=/usr/lib/{target_arch}-linux-gnu/ldscripts");
```

Comment:
I think it might be worthwhile to have receiving those linker script paths from environment variables.

Reply:
Perhaps a `LINKER_SCRIPT` env var with a fallback to the default gnu script location. I think the only real reason you'd want to override this would be if using a musl toolchain.